scripts/deploy_on_kind.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -e
+
+MR_NAMESPACE="${MR_NAMESPACE:-kubeflow}"
+
+if [[ -n "$LOCAL" ]]; then
```

Comment:
```suggestion
# modularity to allow re-use this script against a remote k8s cluster
if [[ -n "$LOCAL" ]]; then
```

I believe it would be easier for people to notice this which is great if we annotate with a small comment here :)